vue/src/components/templates/MainPage.vue

```diff
@@ -68,6 +72,8 @@ export default {
 		logoLink: { type: String, default: '' },
 		hostedByText: { type: String, default: '' },
 		titleText: { type: String, default: '' },
+		firstCheckUtcTime: { type: String, default: '' },
```

Comment:
if it was on me, I'd name those params utc-check-first and utc-check-last, the t from utc means time